packages/atomic/src/components/commerce/atomic-commerce-product-list/atomic-commerce-product-list.tsx

```diff
@@ -263,7 +263,10 @@ export class AtomicCommerceProductList
         this.imageSize
```

Comment:
One annoying thing I've noticed in commerce is that clicking the "show more" button to expand the product description will redirect...

I don't think this needs to be fixed right away, but we should log another bug.